dll/win32/syssetup/wizard.c

```diff
@@ -586,28 +614,31 @@ WriteDefaultLogonData(LPWSTR Domain)
                            KEY_SET_VALUE,
                            &hKey);
     if (lError != ERROR_SUCCESS)
+    {
+        DPRINT1("RegOpenKeyExW for Winlogon failed (%08lX)\n", lError);
         return FALSE;
+    }
 
     lError = RegSetValueEx(hKey,
                            L"DefaultDomainName",
                            0,
                            REG_SZ,
                            (LPBYTE)Domain,
-                           (wcslen(Domain)+ 1) * sizeof(WCHAR));
+                           (wcslen(Domain) + 1) * sizeof(WCHAR));
     if (lError != ERROR_SUCCESS)
     {
-        DPRINT1("RegSetValueEx(\"DefaultDomainName\") failed!\n");
+        DPRINT1("RegSetValueEx(\"DefaultDomainName\") failed (%08lX)\n", lError);
```

Comment:
The current function returns `TRUE`, even when there are failures.
Either fix it or add a comment about that behavior?

Reply:
Fixed.

Reply:
@SergeGautherie  and @katahiromz This function returning FALSE if there is an error with the Registry changes will cause the setup screen to not advance to the next step and there is nothing that the user can do to fix either of these last two error, so I believe that returning TRUE and allowing advancing to the next setup step is the correct behavior here. For an example of what happens just change a character is the registry path to break this and then try building from a clean start and you will see what I mean. Basically it is impossible to install ReactOS under these circumstances. Thanks.